run-clang-tidy.sh

```diff
@@ -47,7 +47,7 @@ cmake -DCMAKE_BUILD_TYPE=Debug \
 [ -a ${CLANG_TIDY_BUILD_DIR}/compile_commands.json ]
 
 # We must populate the includes directory to check things outside of src/
-cd ${CLANG_TIDY_BUILD_DIR} && make HalideIncludes
+cmake --build ${CLANG_TIDY_BUILD_DIR} --target HalideIncludes
```

Comment:
I think the newline ends up with a different working directory than the old one. Everything that follows is absolute paths though, I think....

Did you test this? If so, lgtm.

Reply:
I did test it and it worked locally.